core/Cargo.toml

```diff
@@ -39,24 +39,7 @@ default-members = ["."]
 members = [".", "fuzz", "edge/*", "benches/vs_*"]
 
 [features]
-default = [
-  "rustls",
-  "services-azblob",
-  "services-azdls",
-  "services-cos",
-  "services-fs",
-  "services-gcs",
-  "services-ghac",
-  "services-http",
-  "services-ipmfs",
-  "services-memory",
-  "services-obs",
-  "services-oss",
-  "services-s3",
-  "services-webdav",
-  "services-webhdfs",
-  "services-azfile",
-]
+default = ["rustls"]
```

Comment:
not sure whether it's a good idea to remove `rustls`, so just keep it.

Reply:
Makes sense.